src/brpc/server.cpp

```diff
@@ -2191,6 +2197,21 @@ int Server::MaxConcurrencyOf(google::protobuf::Service* service,
     return MaxConcurrencyOf(service->GetDescriptor()->full_name(), method_name);
 }
 
+bool Server::AcceptRequest(Controller* cntl) const {
+    const Interceptor* interceptor = _options.interceptor;
+    int error_code = 0;
+    std::string error_text;
+    if (cntl && interceptor &&
+        !interceptor->Accept(cntl, error_code, error_text)) {
+        cntl->SetFailed(error_code,
```

Comment:
And the Erron, should it always be predefined Enum EREJECT?

Reply:
Maybe it would be more appropriate to let the user set the error code. Give users more control, such as custom client retry logic.

Reply:
OK